packages/dds/tree/src/feature-libraries/modular-schema/discrepancies.ts

```diff
@@ -404,7 +405,10 @@ export function isRepoSuperset(view: TreeStoredSchema, stored: TreeStoredSchema)
 	for (const incompatibility of incompatibilities) {
 		switch (incompatibility.mismatch) {
 			case "nodeKind": {
-				return false;
+				if (incompatibility.stored !== undefined) {
+					return false;
+				}
+				break;
```

Comment:
As someone who is not familiar with this code, I found myself wishing for a comment that spells out what this logic means. Something like "it's fine for the view schema to know of a node kind that the stored schema doesn't know about" (but hopefully a bit more formal).

Perhaps such a comment would be easier if the code structure were flipped:
```suggestion
				if (incompatibility.stored === undefined) {
					// It's fine for the view schema to know of a node kind that the stored schema doesn't know about
					break;
				}
				return false;
```